src/users_db.rs

```diff
@@ -307,8 +313,10 @@ impl UsersDb {
     /// ```
     pub fn create(&self, user: &User) -> rusqlite::Result<User> {
```

Comment:
The `User` instance already contains the `is_admin` flag that tells the DB whether this should be a creation of an admin user or not. What would be the benefit of separating this into `create` and `createAdmin`?